projects/aca-content/folder-rules/src/rule-list/rule-list/rule-list.ui-component.scss

```diff
@@ -1,26 +1,33 @@
+/* stylelint-disable selector-class-pattern */
+@use '@angular/material' as mat;
+
 .aca-rule-list {
-  display: flex;
-  flex-direction: column;
-  overflow-y: auto;
-  gap: 8px;
+  @include mat.expansion-overrides(
+    (
+      container-background-color: var(--theme-background-color),
+      header-text-color: var(--theme-text-color),
+      container-shape: 12px
+    )
+  );
 
   &__item {
-    display: flex;
-    flex-direction: column;
     border: 1px solid var(--theme-border-color);
     border-radius: 12px;
-    overflow: hidden;
+    /* stylelint-disable-next-line declaration-no-important */
+    box-shadow: none !important;
```

Comment:
Do we need that important? We can try to have higher specificity

Reply:
We could but this would require couple Material selectors usage, this component is not extended anywhere so I think that using important here will be better than material selectors